dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -199,32 +199,34 @@ def load_section
     end
   end
 
-  def select_level
+  def select_levels
     # If there's only one level in this scriptlevel, use that
-    return @script_level.levels[0] if @script_level.levels.length == 1
+    return @script_level.levels if @script_level.levels.length == 1
 
-    # For teachers, load the student's most recent attempt
-    if @user && current_user != @user
-      last_attempt = @user.last_attempt_for_any(@script_level.levels)
-      return last_attempt.level if last_attempt
-    end
-
-    # If they've tried at least one variant before, use the most recently attempted
-    # (unless overridden by a force_reload query string param)
+    # If their most recent attempt was on a now-inactive level, show that.
```

Comment:
The selector won't show an inactive level, so if their most recent activity is on an inactive level, that has to be the only level returned to the client.

If there are multiple active levels (and there are no inactive levels/their most recent activity was not on the inactive level), we'll be showing the selector anyway, so no need to pick just the most recent.